libcodechecker/server/client_db_access_handler.py

```diff
@@ -116,6 +117,30 @@ def process_report_filter_v2(report_filter, count_filter=None):
 
         AND.append(or_(*OR))
 
+    detection_status = report_filter.detectionStatus
+    if report_filter.detectedDate:
+        date = datetime.strptime(report_filter.detectedDate, '%Y-%m-%d')
+
+        OR = []
+        if detection_status is not None and len(detection_status) == 1 and \
+            shared.ttypes.DetectionStatus.RESOLVED in detection_status:
+            OR.append(Report.fixed_at >= date)
+        else:
+            OR.append(Report.detected_at >= date)
+        AND.append(or_(*OR))
+
+    if report_filter.fixedDate:
+        date = datetime.strptime(report_filter.fixedDate, '%Y-%m-%d')
+        tomorrow = date + timedelta(days=1)
```

Comment:
Please use `datetime.timedelta()` if it is possible. At other places, `datetime.strfptime()` is used in a similar manner.

Reply:
If I'm using this way I'm getting the following error:
```bash
type object 'datetime.datetime' has no attribute 'timedelta'
```
I used the following import to resolve this problem:
```python
from datetime import datetime, timedelta
```